doc/pam_hbac.8.txt

```diff
@@ -22,6 +22,8 @@ more details about the configuration file.
 if the user requesting access is not found. If this option is enabled, `pam_hbac`
 will return `PAM_IGNORE` instead, causing the PAM stack to ignore this module.
 
+* *debug* - this option enables verbose logging.
```

Comment:
Once we decide what is the default, the manpage should specify what messages are enabled by default.